lib/tapioca/generator.rb

```diff
@@ -559,6 +565,15 @@ def compile_gem_rbi(gem)
       end
     end
 
+    sig { params(rbi: RBI::Tree).returns(String) }
+    def transform_rbi(rbi)
```

Comment:
What do you think about moving this to the tree here: https://github.com/Shopify/tapioca/blob/main/lib/tapioca/rbi_ext/model.rb.